chart/openfaas/templates/alertmanager-dep.yaml

```diff
@@ -31,6 +31,11 @@ spec:
           - "--config.file=/alertmanager.yml"
           - "--storage.path=/alertmanager"
         livenessProbe:
+          {{- if .Values.httpProbe }}
```

Comment:
I can merge this PR as is, but I wondered if we could get a helm macro to expand this, or is that what you've done?

Reply:
i didn't use a macro, because we generally haven't used them so far and they weren't explicitly asked for

Reply:
I can move it to a macro, if you would like